bitnami/kafka/README.md

```diff
@@ -1084,6 +1084,9 @@ Other notable changes:
 - `externalAccess.autoDiscovery` parameters have been moved under `defaultInitContainers` parameter.
 - `controller.initContainerResources` and `broker.initContainerResources` have been removed. Use `defaultInitContainers.prepareConfig.resources` instead.
 - `extraInit` has been renamed to `defaultInitContainers.prepareConfig.extraInit`.
+- `extraConfig` has been replaced with `overrideConfiguration`.
+- provisioning variable `$CLIENT_CONF` has been removed use `/shared/client.properties`.
```

Comment:
Please remove this line, this was actually done on a follow-up PR, see https://github.com/bitnami/charts/pull/32613